examples/meta/src/evaluation/multiclass_ovr_evaluation.sg.in

```diff
@@ -0,0 +1,27 @@
+File f_feats_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
```

Comment:
this example should also have a file btw

Reply:
kk on it

Reply:
no need to "evaluation" in the example filename (the folder already tells us)